requirements.txt

```diff
@@ -19,6 +19,7 @@ fonttools==4.53.0
 frozenlist==1.4.1
 fsspec==2024.6.0
 fvcore==0.1.5.post20221221
+gradio
```

Comment:
Add dependency using poetry with `poetry add gradio`, which would also update `poetry.lock`; then you could generate an updated `requirements.txt` (if needed) or drop it entirely to avoid potential inconsistencies.